internal/evaluator/builder.go

```diff
@@ -207,18 +207,12 @@ func (bldr *Builder) buildField(
 		Descriptor: fieldDescriptor,
 		Required:   fieldConstraints.GetRequired(),
 		IgnoreEmpty: fieldDescriptor.HasPresence() ||
-			fieldConstraints.GetIgnoreEmpty() ||
-			fieldConstraints.GetIgnore() == validate.Ignore_IGNORE_IF_UNPOPULATED ||
-			fieldConstraints.GetIgnore() == validate.Ignore_IGNORE_IF_DEFAULT_VALUE,
-		IgnoreDefault: fieldDescriptor.HasPresence() && fieldConstraints.GetIgnore() == validate.Ignore_IGNORE_IF_DEFAULT_VALUE,
+			bldr.shouldIgnoreEmpty(fieldConstraints),
```

Comment:
nit: Could put these back on one line if you want now that they're much shorter.

Reply:
kinda like it on multiple lines personally 😅